processors/src/main/java/com/axellience/vuegwt/processors/component/ComponentExposedTypeGenerator.java

```diff
@@ -870,7 +870,7 @@ private void addEmitEventCall(ExecutableElement originalMethod,
       MethodSpec.Builder proxyMethodBuilder, String methodCallParameters) {
     String methodName = "$emit";
     if (methodCallParameters != null && !"".equals(methodCallParameters)) {
-      proxyMethodBuilder.addStatement("vue().$L($S, $L)",
+      proxyMethodBuilder.addStatement("vue().$L($S, jsinterop.base.Js.asAny($L))",
```

Comment:
This is not strictly necessary, and I can remove if you'd prefer. One way or another Js.asAny will be called on this value, either through the other methods that now exist which call Js.asAny, or as this inlined call.

Reply:
I think we can keep the inline call here, there is cleaner way to reference methods in JavaPoet though, but I'll deal with this small change 👍